### ISSUE TITLE
Why are "Run All Tests" and "Run Test" links not showing on test classes in Visual Studio Code?

### ISSUE THREAD

**Issue by xiaozhaozhang** [forcedotcom/salesforcedx-vscode#5192]
Related: [#1675](https://github.com/forcedotcom/salesforcedx-vscode/issues/1675), [#1168](https://github.com/forcedotcom/salesforcedx-vscode/issues/1168)

### Summary

It mentions "Run All Tests" and "Run Test". The problem is that I don't see these links within my Test class. 

### Steps To Reproduce:

1. Open the Test sidebar
2. No Tests found
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/20764296/cc0ba310-a950-4ca1-b7bb-fe4e2366eb92)

### Expected result

Expected tests to be listed, they are not.

### Actual result

No test results found

### Additional information

![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/20764296/d38a3c6c-147b-4de0-836b-227e81688240)

**Salesforce Extension Version in VS Code**:
Salesforce Extension Pack v59.1.2

**SFDX CLI Version**:
@salesforce/cli/2.11.8 win32-x64 node-v18.17.1

**OS and version**:
Windows 11 Enterprise　22H2

**VS Code version**:
Version: 1.81.1 (user setup)
Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794
Date: 2023-08-09T22:22:42.175Z
Electron: 22.3.18
ElectronBuildId: 22689846
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621

**Java version**:
openjdk version "17.0.8" 2023-07-18 LTS
OpenJDK Runtime Environment Zulu17.44+15-CA (build 17.0.8+7-LTS)
OpenJDK 64-Bit Server VM Zulu17.44+15-CA (build 17.0.8+7-LTS, mixed mode, sharing)

**Comment by bishwambharsen** [comment]
Facing the same issue recently. Please help.

**Comment by xiaozhaozhang** [comment]
@daphne-sfdc Thank you for your answer, but I have followed the above steps, the problem is still not solved. Do you have any other solutions?

**Comment by daphne-sfdc** [comment]
Hi @xiaozhaozhang,
Can I see your logs?  There are 2 logs that I would like to see:
1. Run "Developer: Toggle Developer Tools" and go to the Console tab.
2. Open the Output panel and choose "Apex" from the dropdown.
<img width="1465" alt="Screenshot 2023-11-02 at 10 06 34 AM" src="https://github.com/forcedotcom/salesforcedx-vscode/assets/139700604/0fe4e253-95cc-4f85-9788-d785c5d2f8d0">

**Comment by xiaozhaozhang** [comment]
Hi @daphne-sfdc,
I checked the logs of Console tab and "Apex" Output panel and found no problems.
But I saw an error message in the PROBLEMAS tab, which seemed to prompt me that the VsCode version was too low. 
<img width="601" alt="20231103_003" src="https://github.com/forcedotcom/salesforcedx-vscode/assets/20764296/d1a06f44-6537-4d2c-a2ff-69e16d9c4492">
Then I updated VsCode to version 1.84.0, and the problem was solved perfectly.
At the same time, thank you very much for the helpful information and ideas for solving the problem.
